index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <a class="logo" href="#">My Logo</a>
+            
+            <img src="statics/images/Union.png" alt="Menu">
```

Comment:
- [ ] The hamburger icon on the right should be wrapped inside the `<nav>`tag for semantics reasons.

Reply:
Do you mean that I should create new nav  tag that can wrap the humberguer icon and which in his turn wrapped inside the first tag.